src/components/login/index.jsx

```diff
@@ -0,0 +1,172 @@
+import { Alert, Input } from "antd";
+import React, { useState } from "react";
+import { validateEmail } from "../../functions";
+import "./index.css";
+
+const Login = (props) => {
+  const [values, setValues] = useState({ email: "", password: "" });
+
+  const handleChange = (key, value) => {
+    let newValues = Object.assign({}, values);
+    newValues[key] = value;
+    setValues(newValues);
+  };
+  const [errors, setErrors] = useState({});
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    let { email, password } = values;
+    let errors = {};
+    if (!validateEmail(email)) {
+      errors["email"] = "Email format error";
+    }
+    if (email.trim() === "") {
+      errors["email"] = "Please fill this field";
+    }
+
```

Comment:
You may delete extra lines in the function and if you think it looks too messy you can use semicolon after closing curly bracket.

Reply:
I will double-check it.